department-of-veterans-affairs/caseflow
# Fix hearing flake

Resolves [these](https://app.circleci.com/pipelines/github/department-of-veterans-affairs/caseflow/30264/workflows/580ebb03-8e95-4c68-ab58-7f3b85b12f6a/jobs/110706) 5 flakes

### Description
Added missing space, removed blank padding for single digits, ensured gsub does not return nil.

### Acceptance Criteria

### Testing Plan
1. Pull this branch. 
2. Add `Timecop.freeze(Time.zone.now - 3.hours)` [here](https://github.com/department-of-veterans-affairs/caseflow/blob/e6b69d6bdd9741aa9d47ab86b67313d689e85fef/spec/feature/hearings/virtual_hearings/hearing_details_spec.rb#L56)
3. Confirm tests pass when in single digit hours
